docker/etna-base/Makefile

```diff
@@ -2,5 +2,6 @@ baseTag:=$(shell basename "$$(pwd)")
 fullTag:=$(IMAGES_PREFIX)$(baseTag)$(IMAGES_POSTFIX)
 
 image:
+	make -C ../etna-base-dev
```

Comment:
This also fixes an issue where etna-base could be invoked to be built before etna-base-dev, despite having it as a dependency.  I can't specify it as an item level dependency because it is a separate makefile al together, so this command sends a make invocation to it's file.